dm_cli/utils/reference.py

```diff
@@ -22,6 +22,24 @@ def resolve_reference(reference: str, dependencies: Dict[str, Dependency], data_
     return f"dmss://{data_source}/{root_package}/{reference}"
 
 
+def replace_relative_references_in_package_meta(
+    package: Package, dependencies: Dict[str, Dependency], data_source_id: str
+) -> Package:
+    """
+    Replace relative references in meta attribute of the package and subpackages inside the package's content list.
+    Recursively dig down into the package structure and replace the references inside the meta attribute of the package.
+    """
+
+    package.meta = replace_relative_references(
+        "_meta_", package.meta, dependencies, data_source_id, file_path=package.path()
+    )
+
+    for i, document in enumerate(package.content):
+        if type(document) == Package:
+            replace_relative_references_in_package_meta(package.content[i], dependencies, data_source_id)
```

Comment:
I think this is easier to read
```suggestion
    for document in package.content:
        if type(document) == Package:
            replace_relative_references_in_package_meta(document, dependencies, data_source_id)
```